data/mods/No_Hope/Mapgen/mil_base_z0.json

```diff
@@ -1008,7 +1009,7 @@
       [ "mil_base_5j", "mil_base_6j", "mil_base_7j", "mil_base_8j" ],
       [ "mil_base_5k", "mil_base_6k", "mil_base_7k", "mil_base_8k" ]
     ],
-    "weight": 20050,
+    "weight": 25000,
```

Comment:
This is just fixing a typo. Every other 250 is changed to 25000, so I assume this was intended to as well.